aws/v9.3.4/README.md

```diff
@@ -0,0 +1,61 @@
+# :zap: Giant Swarm Release v9.3.4 for AWS :zap:
+
+This release updates managed apps to the latest releases and includes security fixes for 3 CVEs:
+- IPv6 rogue router advertisement vulnerability [CVE-2020-13597](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13597)
+- Intel Microcode vulnerabilities [CVE-2020-0543](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-0543)
+- [CVE-2020-8558](https://github.com/kubernetes/kubernetes/issues/92315) which allows for neighboring hosts to bypass localhost boundary
+
```

Comment:
```suggestion

It also updates managed apps to the latest releases.

```

I saw @sslavic feedback on sig-product and think it makes sense. wdyt @sslavic ?